### PR TITLE
Fix 3rd party component logging

### PR DESCRIPTION
**Pull request by odinuv**
https://keboola.atlassian.net/browse/PS-2945
The searches rely on having `critical` message level - such as `('"component":"kds-team.') ('"priority":"ERR' OR 'priority":"EMERGENCY' OR 'priority":"CRITICAL') -kbc-devel- -'level":400' -'level":401'` e.g. https://my.papertrailapp.com/groups/2307723/

This should log the application error as critical so that we don't have to update all searches.

### CODE REVIEW

File: src/Helper/ExceptionConverter.php [hunk 1]
```diff
@@ -20,21 +20,26 @@ public static function convertExceptionToResult(
         string $jobId,
         array $outputs
     ): JobResult {
+        $transformedException = ExceptionTransformer::transformException($e);
         if (is_a($e, UserException::class)) {
-            $errorTypeString = 'user';
             $errorType = JobResult::ERROR_TYPE_USER;
+            $logger->error(
```
Comment (joker806) [inline src/Helper/ExceptionConverter.php:26]: Hele a nedavalo by ve finale smysl udelat vsehcny ty logy `critical`?

**Review by joker806 — APPROVED** [review]
Jako je to takovy trochu divny :) ale tehcnicky v pohode, takze proc ne..

File: src/Helper/ExceptionConverter.php (hunk 1 above)
Comment (odinuv) [reply src/Helper/ExceptionConverter.php:26]: Ne, protože by pak chodily alerty na user errory :)

**Comment by odinuv** [comment]
to byl omyl

**Pull request merged**